deja_brew/auth/__init__.py

```diff
@@ -0,0 +1,88 @@
+import uuid
+
+import requests
+from flask import Blueprint, request, redirect, current_app, session, abort
+# https://accounts.google.com/.well-known/openid-configuration
+from flask_login import login_user, logout_user
+from oauthlib.oauth2 import WebApplicationClient
+
+from deja_brew.repository import db, User
+
+GOOGLE_AUTH_ENDPOINT = "https://accounts.google.com/o/oauth2/v2/auth"
+GOOGLE_TOKEN_ENDPOINT = "https://oauth2.googleapis.com/token"
+GOOGLE_USER_INFO_ENDPOINT = "https://openidconnect.googleapis.com/v1/userinfo"
+
+auth_bp = Blueprint('auth', __name__)
+
+
+@auth_bp.route('/logout')
+def logout():
+    logout_user()
+    return redirect('/')
+
+
+@auth_bp.route('/login/google')
```

Comment:
bruh, gotta have sign in with apple these days 😩

Reply:
Internal app :D